model/task/task.go

```diff
@@ -604,10 +604,7 @@ func UnscheduleStaleUnderwaterTasks(distroID string) (int, error) {
 		query[DistroIdKey] = distroID
 	}
 
-	query["$and"] = []bson.M{
-		{ActivatedTimeKey: bson.M{"$lte": time.Now().Add(-UnschedulableThreshold)}},
-		{ActivatedTimeKey: bson.M{"$gt": util.ZeroTime}},
```

Comment:
What about inactive tasks? Is there a risk that a task that has not yet been activated will have its priority set to -1 by this PR?

Reply:
The query starts with the result of `scheduleableTasksQuery()` which specifies https://github.com/evergreen-ci/evergreen/blob/f0e7a0ef51cee7b09e00a91dd2fb8396b49186c1/model/task/db.go#L445